pkg/deploy/generator/resources_rp.go

```diff
@@ -1003,6 +1003,16 @@ cat >/etc/default/vsa-nodescan-agent.config <<EOF
   }
 EOF
 
+# we start a cron job to run every hour to ensure the said directory is accessible 
+# by the correct user as it gets created by root and may cause a race condition 
+# where root owns the dir instead of syslog
+# TODO: samann https://github.com/Azure/ARO-RP/issues/1837
```

Comment:
Should we create a work item for us to review the code with a dependency on https://dev.azure.com/msazure/One/_workitems/edit/12512148 instead of github issue?

Reply:
It doesn't matter to me. I am "following" the work item so I should get b- e-mails on activity, but we as a team would probably have increased visibility if we did that.